### PR TITLE
chore: manually create 0.8.0-rc.8

### PR DESCRIPTION
**Pull request by jbadan**
### Description
woops

already deleted tags, waiting on user login from @droshev to be able to delete the published versions on npm.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9551,28 +9551,27 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```
Comment (markpalfreeman) [inline package-lock.json]: I didn't see these changes going from `0.8 -> 1.0`—do you know how they got here going backward?

Comment (jbadan) [reply package-lock.json]: ...because that's another packages dependency?

Comment (jbadan) [reply package-lock.json]: I did a fresh npm install so it probably bumped some stuff. I wouldn't comb through the package-lock manually 😆

Comment (markpalfreeman) [reply package-lock.json]: It's still good to gut-check it.

Yes, probably the new install, I was just curious as these changes alone "shouldn't" affect the lock file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,23 +2,16 @@
...
 
-<a name="1.0.0-rc.1"></a>
-# [1.0.0-rc.1](https://github.com/SAP/fundamental-react/compare/v1.0.0-rc.0...v1.0.0-rc.1) (2019-12-09)
 
-
-
-<a name="1.0.0-rc.0"></a>
-# [1.0.0-rc.0](https://github.com/SAP/fundamental-react/compare/v0.8.0-rc.7...v1.0.0-rc.0) (2019-12-05)
+<a name="0.8.0-rc.8"></a>
+# [1.0.0-rc.0](https://github.com/SAP/fundamental-react/compare/v0.8.0-rc.7...v0.8.0-rc.8) (2019-12-05)
```
Comment (markpalfreeman) [inline CHANGELOG.md]: typo: `[1.0.0-rc.0]` should be switched

**Pull request merged**